src/tests/crud.test.js

```diff
@@ -0,0 +1,63 @@
+import { Task, Store, UI } from '../add_remove_update.js';
+
+// delete window.location;
+// window.location = {
+//     reload: jest.fn()
+// }
+// ? testing the adding function
```

Comment:
- [ ] [REQUIRED] Please, make sure to not include any commented code becasue it pollutes the code base and makes it harder to maintain.